### PR TITLE
[EmailCommunication] multiple Enhancements

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/28837

## Description
This is a part of the full solution, In this PR i added:
1. Classification and mapping for the supported integrations (EWS + Gmail)
2. Added more incident fields and the email communication to existing incident fields (from phishing pack)
3. Changed the original email details section in layout to support inline images.

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/EmailCommunication/IncidentFields/incidentfield-Email_HTML_Image.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (michalgold) [inline Packs/EmailCommunication/IncidentFields/incidentfield-Email_HTML_Image.json:1]: please move to phishing pack

File: Packs/Phishing/IncidentFields/incidentfield-Attachment_ID.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
  "group": 0,
  "hidden": false,
  "associatedTypes": [
-  "Phishing"
+  "Phishing",
+  "Email Communication"
  ],
  "systemAssociatedTypes": [
```
Comment (michalgold) [inline Packs/Phishing/IncidentFields/incidentfield-Attachment_ID.json:30]: In order for the association to work, please add to "systemAssociatedTypes" as well for all other fields as well

Comment (altmannyarden) [reply Packs/Phishing/IncidentFields/incidentfield-Attachment_ID.json:30]: The email communication incident type is already in the "systemAssociatedTypes" for the relevant incident fields

File: Packs/EmailCommunication/Scripts/DisplayEmailHtml/DisplayEmailHtml.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+email_subject = custom_fields.get('emailsubject')
+email_html = custom_fields.get('emailhtml')
+email_html_image = custom_fields.get('emailhtmlimage')
+attachments = incident.get('attachment', {})
+files = demisto.context().get('File', [])
+
+if email_html_image is None:
+    entry_id_list = get_entry_id_list(attachments, files)
+    html_body = create_email_html(email_html, entry_id_list)
+    while 'src="cid' in html_body:
```
Comment (Bargenish) [inline Packs/EmailCommunication/Scripts/DisplayEmailHtml/DisplayEmailHtml.py:93]: @altmannyarden That condition could cause an infinite loop in cases the attachments weren't uploaded successfully to the server. The `create_email_html` function would replace all 'src="cid' instances for attachments that have a relative entry id.

File: Packs/EmailCommunication/Classifiers/classifier-Gmail_-_Incoming_Mapper_-_Email_Communication.json [hunk 4]
```diff
@@ -0,0 +1,419 @@
...
+           "complex": null,
+           "simple": "Headers.Name"
+          }
+         },
+         "operator": "isEqualString",
+         "right": {
+          "isContext": false,
+          "value": {
+           "complex": null,
+           "simple": "Message-Id"
```
Comment (Bargenish) [inline Packs/EmailCommunication/Classifiers/classifier-Gmail_-_Incoming_Mapper_-_Email_Communication.json:176]: Should be "Message-ID"

File: Packs/EmailCommunication/Classifiers/classifier-Gmail_-_Incoming_Mapper_-_Email_Communication.json [hunk 5]
```diff
@@ -0,0 +1,419 @@
...
+     "complex": null,
+     "simple": "Html"
+    },
+    "Email Latest Message": {
+     "complex": {
+      "accessor": "Value",
+      "filters": [
+       [
+        {
+         "ignoreCase": false,
```
Comment (Bargenish) [inline Packs/EmailCommunication/Classifiers/classifier-Gmail_-_Incoming_Mapper_-_Email_Communication.json:163]: @altmannyarden please set the `ignoreCase` to `true`

File: Packs/EmailCommunication/Scripts/DisplayEmailHtml/DisplayEmailHtml.py [hunk 6]
```diff
@@ -0,0 +1,107 @@
+import json
+import re
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def create_email_html(email_html, entry_id_list):
```
Comment (Bargenish) [inline Packs/EmailCommunication/Scripts/DisplayEmailHtml/DisplayEmailHtml.py:8]: @altmannyarden change the function signature to:
`def create_email_html(email_html='', entry_id_list=[]):`

File: Packs/Phishing/ReleaseNotes/1_10_8.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
+
+#### Incident Fields
+- **Email To**
+Added Email Communication to this incident field.
```
Comment (michalgold) [inline Packs/Phishing/ReleaseNotes/1_10_8.md:4]: Incident field association to Email Communication incident type.

**Comment by ShirleyDenkberg** [comment]
@Bargenish @michalgold Doc review completed.

**Review by michalgold — APPROVED** [review]

**Pull request merged**